### ISSUE TITLE
New raptor-browsertime tests instead of old raptor-webext

### ISSUE THREAD

**Issue by klahnakoski** [mozilla-frontend-infra/firefox-health-dashboard#675]
raptor-webext tests for Fenix (Firefox Preview) will be switched off from March 26th now that @gmierz has enabled the raptor-browsertime tests.

**Comment by klahnakoski** [comment]
From sparky: 

The Fenix Browsertime tests were merged into the Fenix repo: https://github.com/mozilla-mobile/fenix/pull/9087
Today, they had their first run: https://treeherder.mozilla.org/#/jobs?repo=fenix&tier=1%2C2%2C3&revision=756b1fbadd7b5aca72167982c600dc540552be04

They are running well, some errors are visible but it's nothing problematic. It looks like we will start turning off Raptor Fenix tests on March 26th/27th.
Note that our "baseline" measures for these various pageload tests will likely change with the move to Browsertime and these changes are unrelated to the application.

Visual metrics will also be running shortly, they are currently blocked on this PR which should land soon: https://github.com/mozilla-mobile/fenix/pull/9248

**Comment by davehunt** [comment]
@airimovici are we ready for the switch from webext to browsertime tomorrow?

**Comment by airimovici** [comment]
> @airimovici are we ready for the switch from webext to browsertime tomorrow?

Working on it. It will be ready for tomorrow
